.npmignore

```diff
@@ -0,0 +1,4 @@
+lib/typescript/example
+**/__tests__
+**/__fixtures__
+**/__mocks__
```

Comment:
I think **/example should be npm ignored as well.

in the package's current state it is publishing the example file and all of the pod dependencies etc.

Reply:
I agree, and this is why we have defined whitelist of files inside of package.json
https://github.com/stripe/stripe-react-native/blob/fix/matching-files-pattern/package.json#L11
combining this with couple of exceptions from .npmignore w get exactly what we need.
@thorsten-stripe @michaelbayday you can run `npm pack` and check what is inside of package.

Reply:
Okay, I now understand that it didn't work because we had negations in the allow list. By moving the negations to the disallow list (.npmignore) the allow list now works as expected 👍